GDJS/GDJS/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -389,10 +396,9 @@ gd::String EventsCodeGenerator::GenerateEventsFunctionContext(
          // Add the new instance to object lists
          "      if (object) {\n" +
          "        objectsList.get(objectsList.firstKey()).push(object);\n" +
-         "        eventsFunctionContext._objectArraysMap[objectName].push(object);\n" +
-         "      }\n" +
-         "      return object;" +
-         "    }\n" +
+         "        "
+         "eventsFunctionContext._objectArraysMap[objectName].push(object);\n" +
+         "      }\n" + "      return object;" + "    }\n" +
```

Comment:
This formatting seems less clear to me, but I guess it was automatic

Reply:
Yes, I'll revert it before merging, i keep it until then so that I do no re-autoformat it by accident

Reply:
I think we could add some "clang-format off" to avoid this :)

Reply:
That wouldn't work since I use almost exclusively the VSCode autoformatter and not clang Format, since most files seem to be formatted with it already. I only use clang format for files I created or where it doesn't too much of the file.